plugins/git/plugin.vala

```diff
@@ -89,6 +90,29 @@ public class Marlin.Plugins.Git : Marlin.Plugins.Base {
             return;
         }
 
+        FileInfo info;
+        try {
+            info = directory.get_target_location ().query_filesystem_info (FileAttribute.FILESYSTEM_TYPE);
```

Comment:
Ideally you can put everything inside the try {} block so that it is clear what it is required for
```suggestion
            var info = directory.get_target_location ().query_filesystem_info (FileAttribute.FILESYSTEM_TYPE);
            if (!info.has_attribute (FileAttribute.FILESYSTEM_TYPE)) {
                debug ("GIT PLUGIN: no filesystem type info - ignoring");
                return;
            }
            
            unowned string fs_type = info.get_attribute_string (FileAttribute.FILESYSTEM_TYPE);
            if (EXCLUDED_FS_TYPES.contains (fs_type)) {
                warning ("GIT PLUGIN: excluded filesystem type %s", fs_type);
                return;
            }
```